lib/src/service/telemetry_websocket_service.dart

```diff
@@ -180,14 +176,20 @@ class TelemetryWebsocketService implements TelemetryService {
     if (_isActive) {
       if (!_isOpened && !_isOpening) {
         _isOpening = true;
-        if (_tbClient.isJwtTokenValid()) {
-          _openSocket(_tbClient.getJwtToken()!);
+        final token = jwtToken();
+        if (token != null) {
+          _openSocket(token);
         } else {
-          _tbClient.refreshJwtToken().then((value) {
-            _openSocket(_tbClient.getJwtToken()!);
+          refreshToken().then((didRefreshSuccessfully) {
```

Comment:
Do you need the boolean flag if you're never returning false?

Reply:
Simple auth gateway refresh token returns boolean future that's doing an API call to Keycloak. I'm just satisfying the Thingsboard client constructor if we want to point it to Thingsboard directly for testing etc.